generators/app/templates/README.md

```diff
@@ -15,9 +15,37 @@
 ## Get things running
 
 * Start the server `npm start`
+* Documentation site available in `/documentation` site
 * Start the server on dev mode `npm run start-dev` 
 * Run tests `npm test`
-* With the server running, open `http://localhost:<%- service.port %>/version` to see the version of your package
 * With the server running, open `http://localhost:<%- service.port %>/documentation` to see the documentation of your API
 
+## Server endpoints
+* GET **`/ops/healthcheck`**: just returns a message saying 'ok'.
+* GET **`/ops/version`**: returns the package.json version.
+<% if(useAuthentication){ -%>
+* POST **`/users/login`**: Login a user and generates a new authentication token which expires in 1 day.
+* POST **`/users/register`**: Sign up - Creates a new user.
+* GET **`/users/me`**: Get the current user information details - [ *Authentication required* ]'
+
+## Authentication flow with JWT
+These are the steps to get a the user information with the `/users/me`, which is the only endpoint that requires authentication:
+
+1. To choose your username and password for the signing-up. To do so, use the `/users/register` endpoint.
+2. To sign-in with your credentials using the endpoint `/users/login`. If the loggin is successful then you will be able to continue the next steps.
+3. To take the **authorization** header received in the server's login response.
+4. To use this authorization header in the GET request of `/users/me`. Finally you will get a response like this:
```

Comment:
Vaya un instructions interruptus:

> To use this header in....

¿Qué? ¿ Qué hay que hacer ? Parece que vas a dar unas instrucciones que luego no das. No querrás decir Use a secas, sin el `To` y las request es algo que apuntas, así que es `request to`